java/vector/src/main/java/org/apache/arrow/vector/ipc/WriteChannel.java

```diff
@@ -41,6 +41,10 @@
 public class WriteChannel implements AutoCloseable {
   private static final Logger LOGGER = LoggerFactory.getLogger(WriteChannel.class);
 
+  private static byte[] zeroBytes = new byte[8];
+
+  private byte[] intBuf = new byte[4];
```

Comment:
This could also be `final`

Reply:
I assume that this whole class is intentionally not thread-safe but it might be worth adding documentation to state that so it is explicit.

Reply:
Revised according to both suggestions. Thank you.